run--01d--prep-acs5-data.Rmd

```diff
@@ -199,9 +202,154 @@ acs5tract_test <-
 #View(acs5tract_test %>% select(wgt, n, r, incpov_r0to50_est, incpov_r0to50_est))
 ```
 
-
 ```{r}
 save(acs5tract, acs5puma,
      file = glue("{output_path}acs5_data_{my_output_tag}.Rda"))
 ```
 
+```{r prepare county argument for fresh pull of acs5 data}
+if (exists("my_county_fip")) county_val <- my_county_fip else county_val <- NULL
+```
+
+```{r fresh pull of both income and work status tables}
+# "Age by Ratio of Income to Poverty Level In the Past 12 Months"
+acs5_incpov_byage_raw    <- pull_table("B17024", state = my_state_abbr, county = county_val, year = acs5_year)
```

Comment:
This does not run when we don't have county_val. Can we add a line -- something like below -- or set this NULL in method--pull-acs5?
`if (exists("my_county_fip")) county_val <- my_county_fip else county_val <- NULL`

Reply:
I don't quite follow -- the `if (exists("my_county_fip"))...` code that you quote is directly above the chunk that you're commenting on. Are you saying that it's not working to avoid errors when `my_county_fip` is not specified? Or that `county_val` somehow isn't being created properly?

Reply:
Ugh.... I have no idea how I missed that. Sorry!